composer.json

```diff
@@ -136,7 +136,8 @@
     "onnov/detect-encoding": "^2.0",
     "knplabs/knp-paginator-bundle": "^6.0.0",
     "masterminds/html5" : "^2.7",
-    "league/flysystem-bundle" : "^3.0",
+    "league/flysystem-bundle": "^3.1",
+    "league/flysystem": "^3.12.0",
```

Comment:
What's the reason for `^3.12.0` ? According to the [docs](https://flysystem.thephpleague.com/docs/usage/checksums/) checksums were added in `3.7.0`

Reply:
@brusch that was also my first attempt to use `3.7.0` version but the tests were [failing](https://github.com/pimcore/pimcore/actions/runs/4604911044/jobs/8136327058). Also they fixed some dependencies in following [versions ](https://github.com/thephpleague/flysystem/blob/3.x/CHANGELOG.md#390---2022-10-18)so i just bumped the latest version.

Reply:
Ok, fine 😊👍